### PR TITLE
Docs/tc-185 Adding links/embedding new tutorial vid/Editing getting started and initial setup guides

### PR DESCRIPTION
**Pull request by NotRobSheridan**
#### Description
Changes made to Welcome and Get started pages. 

Added a link to the new tutorial video from Ollie to the Welcome page and embedded into Get Started. 

the new file `initial-set-up.md` contains the setup info previously found in get-started.md.

initial-set-up.md is now the single source of truth and any new or updated setup info that applies to both the FPS starter project and the GDK in general, should be put into this document. Changes made to initial-set-up.md will be reflected wherever it is included.

#### Tests
Changes tested in Improbadoc local version.

### CODE REVIEW

File: docs/content/get-started/get-started.md [hunk 1]
```diff
@@ -1,8 +1,10 @@
 # Get started: Introduction
 
+<iframe width="100%" height="512" src="https://www.youtube.com/embed/6PNDBYtIimw" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```
Comment (zeroZshadow) [inline docs/content/get-started/get-started.md]: I'm not sure if autoplay is a good idea

File: docs/content/get-started/get-started.md [hunk 2]
```diff
@@ -1,8 +1,10 @@
...
+
 Get started with SpatialOS for Unity by launching a cloud deployment of our FPS Starter Project. This introduces you to the workflows and tooling you use with SpatialOS and Unity, allowing you to run your game at scale, with up to 200 simulated player-clients running in the cloud.
 
-Once you’ve done this, you are then ready to learn how to build your own features. The tutorial for adding Health Pickups functionality introduces you to this development experience.
+Once you have completed this introduction, you are then ready to learn how to build your own features. The [tutorial for adding Health Pickups](https://docs.improbable.io/unity/alpha/projects/fps/tutorial) introduces you to this development experience.
```
Comment (zeroZshadow) [inline docs/content/get-started/get-started.md]: `https://docs.improbable.io/unity/alpha/projects/fps/tutorial` should change too `{{urlRoot}}/projects/fps/tutorial`

File: docs/welcome.md [hunk 3]
```diff
@@ -29,5 +29,9 @@ Looking for a little more inspiration before jumping in?
 <br/>
 * If you’re ready to get started, the _Get started_  guide takes you through setting up the GDK and launching the  FPS Starter Project. (Note that the GDK is aimed at users comfortable with programming in Unity.) <br/>
 **[Get started]({{urlRoot}}/content/get-started/get-started) with the SpatialOS GDK for Unity.**
+<br/>
+<br/>
+* To see just how easy it is to set up the SpatialOS GDK for Unity, follow our **[video tutorial](https://www.youtube.com/watch?v=6PNDBYtIimw) on getting started with SpatialOS for Unity.**
```
Comment (zeroZshadow) [inline docs/welcome.md]: Do we want to link directly here, or should we use some sort of tracking link to view outgoing traffic from this page to the video.

Comment (NotRobSheridan) [reply docs/welcome.md]: Sounds like a good idea. Not sure how to get a tracking link though. @oblm ?

File: docs/content/get-started/get-started.md (hunk 2 above)
Comment (NotRobSheridan) [reply docs/content/get-started/get-started.md]: Fixed. Thanks! :D

File: docs/content/get-started/get-started.md (hunk 1 above)
Comment (NotRobSheridan) [reply docs/content/get-started/get-started.md]: Autoplay removed and related videos set to channel-only. Thanks!

**Comment by NotRobSheridan** [comment]
I have accidentaly made a bunch of changes on this branch that relate to another ticket... Whoops. 

But its all relevant onboarding documentation so not oops?

**Comment by NotRobSheridan** [comment]
screenshot 2019-01-23 at 15 43 51
Per @ElleEss 's request

**Comment by NotRobSheridan** [comment]
Worth noting that I haven't been able to test this on anything other than the local build. I _should_ be able to test the embedding on a live-ish version before the end of the week.

File: docs/setup-and-installing.md [hunk 4]
```diff
@@ -1,137 +1,36 @@
...
+| HTTPS | `git clone https://github.com/spatialos/gdk-for-unity.git` |
 
-**Discord**
+#### Using GitHub Desktop
 
-Find us in the [#unity channel](https://discord.gg/SCZTCYm). You may need to grab Discord [here](https://discordapp.com/).
+To clone the repository using GitHub desktop: navigate to **File > Clone Repository**, click the URL tab and enter `https://github.com/spatialos/gdk-for-unity` into the URL or username/repository field, then click Clone. 
 
-**Github issues**
+#### Checking out the latest release
```
Comment (ElleEss) [inline docs/setup-and-installing.md]:
```suggestion
#### Check out the latest release
```
This one is an instruction if you are using GitHub, I think, so we lose the gerund ("ing").

File: docs/setup-and-installing.md [hunk 5]
```diff
@@ -1,137 +1,36 @@
...
-**Step 4.** Install a **code editor** if you don't have one already
-
-- We recommend either [Visual Studio](https://www.visualstudio.com/downloads/) or [Rider](https://www.jetbrains.com/rider/).
-
-**Using Visual Studio?**
-
-Once you have installed [Visual Studio](https://www.visualstudio.com/downloads/), within the Visual Studio Installer, select **.NET Core + ASP .NET Core**.
-
-**Using Rider?**
+To get set you'll need to create a SpatialOS account, install some dependencies and clone the GDK repository.
```
[the remark this answers is not in the document]
Comment (NotRobSheridan) [reply docs/setup-and-installing.md]: I just spotted this and changed it in the latest commit.

**Comment by NotRobSheridan** [comment]
Made some changes based on @ElleEss 's suggestions.

**Comment by NotRobSheridan** [comment]
Waiting for some changes to the getting started page from Jessica. There will be more commits...

**Comment by NotRobSheridan** [comment]
screenshot 2019-01-25 at 12 08 56

**Comment by NotRobSheridan** [comment]
screenshot 2019-01-25 at 15 58 01
What if that screenshot, but better? 

Changes now available on testing.

File: docs/content/get-started/set-up.md [hunk 6]
```diff
@@ -102,13 +105,17 @@ Unity gives you Mac build support by default.
 
 **Step 3.** Run the **<a href="https://console.improbable.io/installer/download/stable/latest/mac" data-track-link="SpatialOS Installer Downloaded|product=Docs|platform=Mac|label=Mac" target="_blank">SpatialOS installer</a>**
 
-* This installs the [SpatialOS CLI]({{urlRoot}}/content/glossary#spatial-command-line-tool-cli) , the [SpatialOS Launcher]({{urlRoot}}/content/glossary#launcher), and 32-bit and 64-bit Visual C++ Redistributables.
+- This installs the [SpatialOS CLI]({{urlRoot}}/content/glossary#spatial-command-line-tool-cli) , the [SpatialOS Launcher]({{urlRoot}}/content/glossary#launcher), and 32-bit and 64-bit Visual C++ Redistributables.
```
Comment (jessicafalk) [inline docs/content/get-started/set-up.md]:
```suggestion
- This installs the [SpatialOS CLI]({{urlRoot}}/content/glossary#spatial-command-line-tool-cli), the [SpatialOS Launcher]({{urlRoot}}/content/glossary#launcher), and 32-bit and 64-bit Visual C++ Redistributables.
```

File: docs/content/get-started/set-up.md [hunk 7]
```diff
@@ -102,13 +105,17 @@ Unity gives you Mac build support by default.
...
 
 **Step 4.** Install a **code editor** if you don't have one already
 
-* We recommend either [Visual Studio](https://www.visualstudio.com/downloads/) or [Rider](https://www.jetbrains.com/rider/). See _**Code editor set up**_, below.
+- We recommend either [Visual Studio](https://www.visualstudio.com/downloads/) or [Rider](https://www.jetbrains.com/rider/). See _**Code editor set up**_, below.
+
+**Step 5.** Install Git
+
+The SpacialOS GDK for Unity source code is hosted on GitHub. You need to download and install  [Git](https://git-scm.com/downloads) or [GitHub Desktop](https://desktop.github.com/) in order to clone the GDK repositories.
```
Comment (jessicafalk) [inline docs/content/get-started/set-up.md]:
```suggestion
The SpatialOS GDK for Unity source code is hosted on GitHub. You need to download and install [Git](https://git-scm.com/downloads) or [GitHub Desktop](https://desktop.github.com/) in order to clone the GDK repositories. 
```

File: docs/content/get-started/set-up.md [hunk 8]
```diff
@@ -186,22 +192,20 @@ From the root of the `gdk-for-unity-fps-starter-project` repository:
...
-  ```text
+```
   <common_parent_directory>
       ├── gdk-for-unity-fps-starter-project
       ├── gdk-for-unity
-  ```
+```
 
-2. Navigate to the `gdk-for-unity` directory and checkout the pinned version which you can find in the `gdk.pinned` file, in the root of the `gdk-for-unity-fps-starter-project` directory.
+2\.  Navigate to the `gdk-for-unity` directory and checkout the pinned version which you can find in the `gdk.pinned` file, in the root of the `gdk-for-unity-fps-starter-project` directory.
```
Comment (jessicafalk) [inline docs/content/get-started/set-up.md]: out of curiosity: what's that backslash for?

File: docs/content/initial-set-up.md [hunk 9]
```diff
@@ -0,0 +1,138 @@
...
+
+- This installs the [SpatialOS CLI]({{urlRoot}}/content/glossary#spatial-command-line-tool-cli) , the [SpatialOS Launcher]({{urlRoot}}/content/glossary#launcher), and 32-bit and 64-bit Visual C++ Redistributables
+
+**Step 4.** Install a **code editor** if you don't have one already
+
+- We recommend either [Visual Studio 2017](https://www.visualstudio.com/downloads/) or [Rider](https://www.jetbrains.com/rider/). See _**Code editor set up**_, below.
+
+**Step 5.** Install Git
+
+The SpacialOS GDK for Unity source code is hosted on GitHub. You need to download and install  [Git](https://git-scm.com/downloads) or [GitHub Desktop](https://desktop.github.com/) in order to clone the GDK repositories.
```
Comment (jessicafalk) [inline docs/content/initial-set-up.md]:
```suggestion
The SpatialOS GDK for Unity source code is hosted on GitHub. You need to download and install [Git](https://git-scm.com/downloads) or [GitHub Desktop](https://desktop.github.com/) in order to clone the GDK repositories. 
```

:D

File: docs/content/initial-set-up.md [hunk 10]
```diff
@@ -0,0 +1,138 @@
...
+
+- This installs the [SpatialOS CLI]({{urlRoot}}/content/glossary#spatial-command-line-tool-cli) , the [SpatialOS Launcher]({{urlRoot}}/content/glossary#launcher), and 32-bit and 64-bit Visual C++ Redistributables.
+
+**Step 4.** Install a **code editor** if you don't have one already
+
+- We recommend either [Visual Studio](https://www.visualstudio.com/downloads/) or [Rider](https://www.jetbrains.com/rider/). See _**Code editor set up**_, below.
+
+**Step 5.** Install Git
+
+The SpacialOS GDK for Unity source code is hosted on GitHub. You need to download and install  [Git](https://git-scm.com/downloads) or [GitHub Desktop](https://desktop.github.com/) in order to clone the GDK repositories.
```
Comment (jessicafalk) [inline docs/content/initial-set-up.md]:
```suggestion
The SpatialOS GDK for Unity source code is hosted on GitHub. You need to download and install [Git](https://git-scm.com/downloads) or [GitHub Desktop](https://desktop.github.com/) in order to clone the GDK repositories. 
```

File: docs/content/get-started/set-up.md [hunk 11]
```diff
@@ -48,10 +49,13 @@ Unity gives you Windows build support by default.
 
 - We recommend either [Visual Studio 2017](https://www.visualstudio.com/downloads/) or [Rider](https://www.jetbrains.com/rider/). See _**Code editor set up**_, below.
 
-**Step 5.** Restart your machine.
+**Step 5.** Install Git
 
-> If you do not restart your machine, you may experience errors when opening a GDK project.
+The SpacialOS GDK for Unity source code is hosted on GitHub. You need to download and install  [Git](https://git-scm.com/downloads) or [GitHub Desktop](https://desktop.github.com/) in order to clone the GDK repositories.
```
[the remark this answers is not in the document]
Comment (NotRobSheridan) [reply docs/content/get-started/set-up.md]: Yeah good call. We mention it later on in the ## Get the GDK and the FPS Starter Project source code section, but its definitely worth mentioning here too.

File: docs/setup-and-installing.md [hunk 12]
```diff
@@ -38,22 +40,31 @@ Unity gives you Windows build support by default.
 
 **Step 4.** Install a **code editor** if you don't have one already
 
-- We recommend either [Visual Studio 2017](https://www.visualstudio.com/downloads/) or [Rider](https://www.jetbrains.com/rider/).
+- We recommend either [Visual Studio 2017](https://www.visualstudio.com/downloads/) or [Rider](https://www.jetbrains.com/rider/). See _**Code editor set up**_, below.
 
-**Using Visual Studio?**
+**Step 5.** Install Git
+
+The SpacialOS GDK for Unity source code is hosted on GitHub. You need to download and install  [Git](https://git-scm.com/downloads) or [GitHub Desktop](https://desktop.github.com/) in order to clone the GDK repositories.
```
Comment (jessicafalk) [inline docs/setup-and-installing.md]:
```suggestion
The SpatialOS GDK for Unity source code is hosted on GitHub. You need to download and install [Git](https://git-scm.com/downloads) or [GitHub Desktop](https://desktop.github.com/) in order to clone the GDK repositories. 
```

File: docs/content/get-started/set-up.md (hunk 8 above)
Comment (NotRobSheridan) [reply docs/content/get-started/set-up.md]: I was having trouble getting this list to format correctly, it kept creating a new <ol> element for each numbered point so we'd have two 1. steps.  If you have a look at the live docs, it currently has step 1. twice. 

The slash just stops it from creating an <ol> and keeps it as a paragraph.

File: docs/setup-and-installing.md [hunk 13]
```diff
@@ -86,11 +98,21 @@ Unity gives you Mac build support by default.
...
+- This installs the [SpatialOS CLI]({{urlRoot}}/content/glossary#spatial-command-line-tool-cli) , the [SpatialOS Launcher]({{urlRoot}}/content/glossary#launcher), and 32-bit and 64-bit Visual C++ Redistributables.
 
 **Step 4.** Install a **code editor** if you don't have one already
 
-- We recommend either [Visual Studio](https://www.visualstudio.com/downloads/) or [Rider](https://www.jetbrains.com/rider/).
+- We recommend either [Visual Studio](https://www.visualstudio.com/downloads/) or [Rider](https://www.jetbrains.com/rider/). See _**Code editor set up**_, below.
+
+**Step 5.** Install Git
+
+The SpacialOS GDK for Unity source code is hosted on GitHub. You need to download and install  [Git](https://git-scm.com/downloads) or [GitHub Desktop](https://desktop.github.com/) in order to clone the GDK repositories.
```
Comment (jessicafalk) [inline docs/setup-and-installing.md]:
```suggestion
The SpatialOS GDK for Unity source code is hosted on GitHub. You need to download and install [Git](https://git-scm.com/downloads) or [GitHub Desktop](https://desktop.github.com/) in order to clone the GDK repositories. 
```

File: docs/setup-and-installing.md [hunk 14]
```diff
@@ -102,36 +124,48 @@ Once you have installed [Rider](https://www.jetbrains.com/rider/), install the [
...
 
-**Github issues**
+To run the GDK for Unity, you need to download the source code. To do this, you need to clone the GDK repository.
 
-Create an [issue](https://github.com/spatialos/gdk-for-unity/issues) in this repository.
-<%(/Expandable)%>
+#### Using a terminal:
 
-### Clone the SpatialOS GDK for Unity repository
+Clone the SpacialOS GDK for Unity repository using one of the following terminal commands:
```
Comment (jessicafalk) [inline docs/setup-and-installing.md]:
```suggestion
Clone the SpatialOS GDK for Unity repository using one of the following terminal commands:
```

could we also reverse the ordering of SSH and HTTPS? the recommended one is HTTPS

File: docs/setup-and-installing.md [hunk 15]
```diff
@@ -102,36 +124,48 @@ Once you have installed [Rider](https://www.jetbrains.com/rider/), install the [
...
 
+#### Using GitHub Desktop
+
+To clone the repository using GitHub desktop: navigate to **File > Clone Repository**, click the URL tab and enter `https://github.com/spatialos/gdk-for-unity` into the URL or username/repository field, then click Clone. 
+
+#### Check out the latest release
+
+Once you have cloned the repository, you need to checkout the latest release by using the following terminal command: 
+
+`git checkout 0.1.3`
```
Comment (jessicafalk) [inline docs/setup-and-installing.md]: as discussed via slack: this section can go away 🎉

File: docs/content/get-started/get-started.md [hunk 16]
```diff
@@ -2,14 +2,28 @@
 
 Get started with SpatialOS for Unity by launching a cloud deployment of our FPS Starter Project. This introduces you to the workflows and tooling you use with SpatialOS and Unity, allowing you to run your game at scale, with up to 200 simulated player-clients running in the cloud.
 
-Once you’ve done this, you are then ready to learn how to build your own features. The tutorial for adding Health Pickups functionality introduces you to this development experience.
+<iframe width="560" height="315" src="https://www.youtube.com/embed/6PNDBYtIimw?autoplay=0&rel=0" frameborder="0"  allowfullscreen, style="; float: right; margin: 10px 10px 30px 30px; " ></iframe>
 
-We recommend starting with the FPS Starter Project by going to the next step: [Set up]({{urlRoot}}/content/get-started/set-up.md), but you can get going with the [Blank Starter Project]({{urlRoot}}/projects/blank/overview) if you prefer this route.
+### Get started video
+Found out whats inlvolved in getting started with our **Get started with SpatialOS for Unity walk-through youtube video**, based on our Get Started guide (15-minute watch). <br/>
```
Comment (jessicafalk) [inline docs/content/get-started/get-started.md]:
```suggestion
Found out what's involved in getting started with our **Get started with SpatialOS for Unity walk-through youtube video**, based on our Get Started guide (15-minute watch). <br/>
```

File: docs/content/get-started/get-started.md [hunk 17]
```diff
@@ -2,14 +2,28 @@
...
+We recommend starting with the FPS Starter Project by going to the next step: [Set up]({{urlRoot}}/content/get-started/set-up.md), but you can get going with the [Blank Starter Project]({{urlRoot}}/projects/blank/overview) if you prefer this route.
 <br/>
+
+> **Note:** Whichever route you choose, you need basic proficiency in [programming in Unity](https://unity3d.com/programming-in-unity), version control systems (such as [Git](https://try.github.io/)), and [command line interfaces](https://tutorial.djangogirls.org/en/intro_to_command_line/) to use the SpatialOS GDK for Unity.
+
+#### Next: [Set up]({{urlRoot}}/content/get-started/set-up.md)
+
 ![Instant action in the FPS Starter Project]({{assetRoot}}assets/fps/headline.png)
 
+
```
Comment (jessicafalk) [inline docs/content/get-started/get-started.md]: are these newlines needed?

File: docs/content/get-started/set-up.md [hunk 18]
```diff
@@ -124,23 +135,25 @@ Once you have installed [Rider](https://www.jetbrains.com/rider/), install the [
...
-* Unity gives you build support for your development machine (Windows or Mac) by default.
 
+- You need **Linux** build support. This is because server-workers in a cloud deployment run in a Linux environment.<br/>
+- You need **Mac** build support if you are developing on a Windows PC and want to share your game with Mac users.<br/>
+- You need **Windows** build support if you are developing on a Mac and want to share your game with Windows PC users. <br/>
+- Unity gives you build support for your development machine (Windows or Mac) by default.
+
+#### Android and iOS support
+
+Mobile support is in pre-alpha. If you are developing a game for Android or iOS, refer to our GDK for Unity [mobile support documentation]({{urlRoot}}/content//mobile/overview)
```
Comment (jessicafalk) [inline docs/content/get-started/set-up.md]: super-nit: 
```suggestion
Mobile support is in pre-alpha. If you are developing a game for Android or iOS, refer to our GDK for Unity [mobile support documentation]({{urlRoot}}/content/mobile/overview)
```

File: docs/setup-and-installing.md [hunk 19]
```diff
@@ -102,36 +128,45 @@ Once you have installed [Rider](https://www.jetbrains.com/rider/), install the [
...
+- You need **Linux** build support. This is because server-workers in a cloud deployment run in a Linux environment.<br/>
+- You need **Mac** build support if you are developing on a Windows PC and want to share your game with Mac users.<br/>
+- You need **Windows** build support if you are developing on a Mac and want to share your game with Windows PC users. <br/>
+- Unity gives you build support for your development machine (Windows or Mac) by default.
 
-Visit the [support section](https://forums.improbable.io/new-topic?category=Support&tags=unity-gdk) in our forums and use the unity-gdk tag.
+#### Android and iOS support
 
-**Discord**
+Mobile support is in pre-alpha. If you are developing a game for Android or iOS, refer to our GDK for Unity [mobile support documentation]({{urlRoot}}/content//mobile/overview)
```
Comment (jessicafalk) [inline docs/setup-and-installing.md]: I wonder if we should also add the warning to the part of the get started guide where we actually build the workers. As that is the part where people might see errors that they did not expect.

Comment (NotRobSheridan) [reply docs/setup-and-installing.md]: We absolutely should. I just totally forgot to do it. Adding now.

File: docs/content/get-started/set-up.md [hunk 20]
```diff
@@ -186,22 +199,20 @@ From the root of the `gdk-for-unity-fps-starter-project` repository:
 
 <%(#Expandable title="Manually clone the GDK for Unity")%>
 
-1. Clone the [GDK for Unity](https://github.com/spatialos/gdk-for-unity) repository alongside the FPS Starter Project so that they sit side-by-side:
-
+1\.  Clone the [GDK for Unity](https://github.com/spatialos/gdk-for-unity) repository alongside the FPS Starter Project so that they sit side-by-side:
```
Comment (jessicafalk) [inline docs/content/get-started/set-up.md]: Only saw it now, but there is a line in this doc saying:
`Then navigate to the root folder of the FPS starter project and run the following command: git checkout 0.1.3`

They don't need to do this anymore. They can use the master branch directly

File: docs/setup-and-installing.md (hunk 19 above)
Comment (jessicafalk) [reply docs/setup-and-installing.md]: let me know when you added it. We would like to get that PR in today for the release

File: docs/content/get-started/set-up.md (hunk 20 above)
Comment (NotRobSheridan) [reply docs/content/get-started/set-up.md]: Removed this like

File: docs/setup-and-installing.md (hunk 19 above)
Comment (NotRobSheridan) [reply docs/setup-and-installing.md]: Done

File: docs/content/get-started/get-started.md (hunk 17 above)
Comment (NotRobSheridan) [reply docs/content/get-started/get-started.md]: Nope. Removed.

**Pull request merged**